src/storage/v2/inmemory/storage.cpp

```diff
@@ -153,8 +153,6 @@ InMemoryStorage::InMemoryStorage(Config config, StorageMode storage_mode)
   }
 }
 
-InMemoryStorage::InMemoryStorage(Config config) : InMemoryStorage(config, StorageMode::IN_MEMORY_TRANSACTIONAL) {}
-
```

Comment:
IN_MEMORY_TRANSACTIONAL is the default value of Config so the behaviour is the same.